contracts/bridge/contracts/binance/BasicInboundChannel.sol

```diff
@@ -27,18 +54,73 @@ contract BasicInboundChannel {
         uint32 blockNumber;
     }
 
-    event MessageDispatched(uint64 nonce, bool result);
+    /**
+     * @notice Notifies an observer that the message has dispatched
+     * @param nonce The message nonce
+     * @param result The message result
+     * @param returndata The return data of message call, when return false, it's the reason of the error
+     */
+    event MessageDispatched(uint256 indexed nonce, bool indexed result, bytes returndata);
+
+    /* State */
+
+    /**
+     * @dev The position of the leaf in the chain message merkle tree, index starting with 0
```

Comment:
The document does not match

Reply:
https://www.notion.so/itering/Basic-Message-Channel-c41f0c9e453c478abb68e93f6a067c52#9607848fcd674196809f060470642707